infrastructure/Dockerfile

```diff
@@ -40,24 +40,26 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-ENV SUPERVISORD_APP_VERSION       3.2.3
+ENV SUPERVISORD_APP_VERSION       3.3.3
 ENV DOCKER_APP_VERSION            17.05.0~ce-0~ubuntu-xenial
-ENV CONSUL_APP_VERSION            0.8.5
-ENV CONSUL_TEMPLATE_APP_VERSION   0.18.2
+ENV CONSUL_APP_VERSION            0.9.3
+ENV CONSUL_TEMPLATE_APP_VERSION   0.19.3
```

Comment:
please dont raise the one - consul template will not work
seems the configuration has changed recently.